pages/docs/concepts/server.md

```diff
@@ -0,0 +1,32 @@
+
+## What is a server?
+A server is a representation of a messaging broker system where connections and communication between a producer and a consumer is established. Unlike traditional API servers which are dependent on request/response, message broker interactions occur back and forth over different channels.
+
+## What is the purpose of servers?
+Servers play an important role as they maintain a relationship between producers and consumers. When designing and setting up an event-driven application, servers are in charge of delivering asynchronous messages from the producer to the consumers through the use of channels. With the integration of different messaging protocols, servers can transmit and exchange messages between clients.
+
+* Clients and Server
+```mermaid
+flowchart TD
+    a[Client Browser] --> b[(server)]
+    b --> a
+    c[Client Mobile] --> b[(server)]
+    b --> c 
+```
+The diagram above describes a bi-directional relationship between `clients` and `server`. When we create an AsyncAPI `server` file, the AsyncAPI server object holds information that includes the physical location of the `server`.
+
+
+* Broker Centric
+```mermaid
+flowchart TD
+    A[producer]
+    A --> a1[channel1]
+    A --> a2[channel2]
+    subgraph one[broker]
+    a1
+    a2
+    end
+    a1 --> B[consumer1]
+    a2 --> C[consumer2]
+```
+The above diagram shows the *Broker Centric Architecture*. We create 3 AsyncAPI document for `producer`, `consumer1`, and `consumer2`. The information provided for these AsyncAPI files comes directly from the `broker` which is a representation of a `server`.
```

Comment:
>  The information provided for these AsyncAPI files comes directly from the `broker` which is a representation of a `server`.

I find this sentence a bit confusing. I think we want to mention that the `Server Object`, in this case, holds information about the **broker**. If that wasn't the case, I would like to understand the meaning (not saying is wrong really, I just don't get it).

Reply:
Hey, @smoya, I'm trying to emphasize here that the server object holding the information is actually the `broker`.  So instead of mentioning the `server`, we want to focus on the `broker`. (I might be getting it wrong though)

Reply:
Yeah, I think we are aligned then 👍 Then I still think it needs a rewording as I suggested; the info doesn't "come" from the broker, but the Broker info is what the Server Object holds, right?

Reply:
Would it make sense if we wrote it like this;

*The above diagram shows the Broker Centric Architecture. We create 3 AsyncAPI documents for the `producer`, `consumer1`, and `consumer2`. The `broker` holds the information provided in the AsyncAPI files.*

Reply:
> The broker holds the information provided in the AsyncAPI files.

I still don't get what this sentence means. Maybe the concept of broker wasn't clear (excuse me if it is not the case 🙏 ). Here we are referring to a [message broker](https://en.wikipedia.org/wiki/Message_broker), which can be seen as a common server other applications connect to in order to communicate between them, i.e. send/receive messages.
 
On AsyncAPI, the broker is represented as a [server object](https://www.asyncapi.com/docs/reference/specification/v2.4.0#serverObject) under `servers` field, in your AsyncAPI document, which is going to be the same in all documents (`producer`, `consumer1`, and `consumer2`). So technically, the broker (remember, a physical server other applications connect to) doesn't know about the AsyncAPI document, but the other way around. Meaning, that server object written under `servers` field, in your AsyncAPI document, shows info about how to connect to that broker.